lib/datasets.go

```diff
@@ -32,44 +32,29 @@ import (
 	"github.com/qri-io/qri/resolver/loader"
 )
 
-// DatasetRequests encapsulates business logic for working with Datasets on Qri
-// TODO (b5): switch to using an Instance instead of separate fields
-type DatasetRequests struct {
+// DatasetMethods encapsulates business logic for working with Datasets on Qri
+type DatasetMethods struct {
 	// TODO (b5) - remove cli & node fields in favour of inst accessors:
 	cli  *rpc.Client
 	node *p2p.QriNode
 	inst *Instance
 }
 
 // CoreRequestsName implements the Requets interface
-func (DatasetRequests) CoreRequestsName() string { return "datasets" }
+func (DatasetMethods) CoreRequestsName() string { return "datasets" }
```

Comment:
Could also like if we could change this method name to CoreMethodsName, though that also touches a lot of files

Reply:
totally agreed, I think it's worth a follow-up PR